src/test/java/io/vertx/core/eventbus/LocalEventBusTest.java

```diff
@@ -52,7 +52,7 @@ public void setUp() throws Exception {
     running = true;
   }
 
-  protected void tearDown() throws Exception {
+  @Override protected void tearDown() throws Exception {
```

Comment:
Please configure the formatter of your IDE or text editor. The annotation should be on a separate line above.